code_challenges/adhoc/two_sum.py

```diff
@@ -0,0 +1,14 @@
+def two_sum(numbers, target):
+
+    for i in range(len(numbers)):
+        for j in range(1, len(numbers)):
```

Comment:
This is an O(n^2) solution.  Which is fine to come up with first.  Can you think of a way to do this in single loop over the list?